### DISCUSSION TITLE
Raspberry Pi Pico | Pico Bricks | Total Newbie

### DISCUSSION THREAD

**Discussion by ikevinax** [micropython/micropython#11345] [General]
I understand that I might be raked over the coals :) However,

When it was fresh out of the box, it had all of these cool features to demonstrate the bricks' capabilities. Now I messed it up somehow. I reloaded the pre-installed files, but it doesn't work.

I have very little idea what I'm doing. I'm going to set the Pico Bricks aside for a moment and try not to break my Adafruit Circuit Playground Express.

Any ideas on exactly how to achieve a "factory reset"?

**Comment by rkompass** [comment]
As for the Adafruit Circuit Playground Express I think a "factory reset" is synonymous to "install circuitpython" for which you find instructions [here](https://learn.adafruit.com/adafruit-circuit-playground-express/circuitpython-quickstart).
Note that Circuitpython is a dialect of Micropython that aims to be very user friendly regarding peripherals. These are accessed differently, and for questions regarding Circuitpython you better user their [forum](https://forums.adafruit.com/viewforum.php?f=60).

PicoBrics seem to use ordinary Micropython (MP). If you have questions regarding that you are at the right place.

They also offer using the Scratch-like MicroBlocks environment. That probably is not discussed here (at least I have not noted any discussion regarding it here), unless there is a possibility to extract the generated MP code.
Also MicroBlocks seems to install a special firmware onto the Pico which probably is not known here. Perhaps they have their own forum too.

**Reply by ikevinax** [reply]
Thanks for your reply. I finally got the original code reinstalled and working. Evidently, I needed to reflash the memory and the original code started working again. I had added the four files, but it wouldn't run them until I did this. Another thing - the Pico that I had temporarily replaced it with did not have WiFi, so some of the code was unfamiliar to it. Thanks again.
